surfnperf.js

```diff
@@ -356,6 +356,26 @@
     return this.duration('navigationStart', 'loadEventEnd');
   };
 
+  /**
+   * Time to first paint (amount of time that lapses before some useful content is diplayed indicating that the browser is not stuck and is still working)
+   *
+   * @returns {integer} time in ms
+   * @memberOf SurfNPerf
+   */
+  SNPProto.getFirstPaint = function() {
+    if(window.chrome && window.chrome.loadTimes) {
+      return window.chrome.loadTimes().firstPaintTime;
```

Comment:
let's actually make this the value relative to when the page started loading, so:
`window.chrome.loadTimes().firstPaintTime*1000 - window.performance.timing.navigationStart`